doc/en/user/source/styling/sld/working.rst

```diff
@@ -20,12 +20,17 @@ Catalog Styles
 Styles in the catalog can be viewed, edited and validated via the :ref:`styling_webadmin_styles` menu of the :ref:`web_admin`. 
 They may also be created and accessed via the REST :ref:`rest_api_styles` API.
 
-Catalog styles consist of a :ref:`sld_reference_sld` document 
+There are two types of Catalog Styles: Layer Styles (the default) and Style Groups.
+
+Layer Styles
+~~~~~~~~~~~~
```

Comment:
While that is definitely more correct, and clearer if you understand what it means, I am a bit concerned that it is more confusing from a user perspective.